README.md

```diff
@@ -190,8 +190,10 @@ api.MetricsRetrieveCustomerChurnRate(&MetricsFilter{})
 api.MetricsRetrieveMRRChurnRate(&MetricsFilter{})
 api.MetricsRetrieveLTV(&MetricsFilter{})
 
-api.MetricsListSubscriptions(&Cursor{}, "customerUUID")
-api.MetricsListActivities(&Cursor{}, "customerUUID")
+api.MetricsListCustomerSubscriptions(&Cursor{}, "customerUUID") // >= v2.0.0
```

Comment:
What does the comment mean? I don't get it.

Reply:
It just means that is the method call is available in version >= v2.0.0. In case someone looks at the read me but is using the package at 1.x then that line will not work for them. Of course if they are looking at the right documentation from package and not the github readme, they will not have issues. I thought maybe it will help. I can remove it if it is just causing more confusion.